unitTests/ui/ValueElementTest.ts

```diff
@@ -42,6 +43,8 @@ export function ValueElementTest() {
       facet = Mock.optionsComponentSetup<Facet, IFacetOptions>(Facet, {
         field: '@field'
       }).cmp;
+      facet.ensureDom();
+      facet.facetSearch = Mock.mock(FacetSearch);
```

Comment:
To remember in the future: if `ensureDom` isn't called before setting the mock, the mock may get overwritten later...